matplotlib-tests/meta.yaml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: matplotlib-tests
+  version: 1.5.0
+
+source:
+  git_url: https://github.com/matplotlib/matplotlib.git
+  git_rev: v1.5.x
+
+requirements:
+  build:
+    - matplotlib ==1.5.0
```

Comment:
Interesting.  I've updated this to use jinja to avoid the version number duplication.  It's still not perfect, since now the package version is defined in a non-standard way (not just in the `package.version` YAML value)...